src/test/kotlin/uk/gov/justice/digital/hmpps/offenderevents/integration/PollCommunityApiTest.kt

```diff
@@ -183,7 +286,7 @@ class PollCommunityApiTest : IntegrationTestBase() {
 
       offenderUpdatePollService.pollForOffenderUpdates()
 
-      await untilCallTo { getNumberOfMessagesCurrentlyOnQueue() } matches { it == 1 }
+      await untilCallTo { getNumberOfMessagesCurrentlyOnQueue() } matches { it == 1 * 2 }
```

Comment:
:-)

Reply:
Have a variable / contant to remove this magic number?